ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthServer.java

```diff
@@ -47,20 +47,30 @@
 class EthServer {
   private static final Logger LOG = LogManager.getLogger();
 
+  static final int DEFAULT_REQUEST_LIMIT = 200;
   private final Blockchain blockchain;
   private final WorldStateArchive worldStateArchive;
   private final EthMessages ethMessages;
-  private final int requestLimit;
+  private final int maxGetBlockHeaders;
+  private final int maxGetBlockBodies;
+  private final int maxGetReceipts;
+  private final int maxGetNodeData;
 
   EthServer(
```

Comment:
How about adding another constructor that uses the default request size values, so we don't need to add all of these new constructor parameters at higher levels?

Reply:
fixed